### PR TITLE
Modified instructions and tutorials

### PR DESCRIPTION
**Pull request by jsantiago-eProsima**
This PR modifies instructions and tutorials to take into account the modified names, paths and dependencies resulting from both the change to Humble and the OS upgrade to Ubuntu Jammy.

Signed-off-by: Javier Santiago <javiersantiago@eprosima.com>

### CODE REVIEW

File: docs/rst/tutorials/cloud_tutorial.rst [hunk 1]
```diff
@@ -127,7 +127,7 @@ DDS-Router deployment
 ^^^^^^^^^^^^^^^^^^^^^
 
 The cloud router is launched from within a *Vulcanexus Cloud* Docker image (that can be downloaded in `Vulcanexus webpage <TODO: include final URL>`_), which uses as configuration file the one hosted in the previously set up ConfigMap.
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/tutorials/cloud_tutorial.rst:129]: This TODO has to be done before release

File: docs/rst/platforms/platforms.rst [hunk 2]
```diff
@@ -14,6 +14,9 @@ Vulcanexus ROS 2 all-in-one tool set, is officially available in the platforms s
     * - :ref:`Galactic Gamble <notes_galactic_latest>`
       - amd64
       - Ubuntu Focal (20.04)
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/platforms/platforms.rst:16]: If there is not going to be a Vulcanexus Galactic release, I think we should delete all related information to Vulcanexus Galactic.

File: docs/rst/notes/notes.rst [hunk 3]
```diff
@@ -27,4 +28,8 @@ The following table outlines the Vulcanexus releases and their support cycles:
      - v1
      - TODO
      - November 2022
+   * - :ref:`Humble Hierro <notes_humble_latest>`
+     - v1
+     - TODO
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/notes/notes.rst:33]: This TODO is May 2022 (it is intended to be launched today or tomorrow)

File: docs/rst/notes/notes.rst [hunk 4]
```diff
@@ -27,4 +28,8 @@ The following table outlines the Vulcanexus releases and their support cycles:
      - v1
      - TODO
      - November 2022
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/notes/notes.rst:30]: Galactic release should be removed because it has not been done yet.

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]
Review without rerunning the instructions, only looking into the changes.

File: docs/rst/platforms/platforms.rst (hunk 2 above)
Comment (jsantiago-eProsima) [reply docs/rst/platforms/platforms.rst:16]: Vulcanexus Galactic should be available at the same time Humble will be.

**Review by JLBuenoLopez-eProsima — COMMENTED** [review]
LGTM, though the CI is still complaining about some spelling

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
:+1: 

Release notes will be included in another PR

**Pull request merged**